src/components/Mission/index.jsx

```diff
@@ -21,8 +21,11 @@ class Mission extends Component {
   render() {
     return (
       <Wrapper>
-          <Heading>Mission Statement</Heading>
-          <Text>We're just trying to make the world a better place and come up with great mission statements!</Text>
+        <Heading>Mission Statement</Heading>
+        <Text>
+          We&apos;re just trying to make the world a better place and come up with
```

Comment:
Making note of another apostrophe replaced with ASCII. **About** had similar note.

Reply:
Yep!